apps/antalmanac/tests/schedule.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { describe, test, expect } from 'vitest';
+import { render } from '@testing-library/react';
+import { Schedules } from '../src/stores/Schedules';
+import RenameScheduleDialog from '$components/dialogs/RenameSchedule';
+
+describe('schedule logic', () => {
+    const scheduleStore = new Schedules();
+
+    test('no error when loading undefined schedule', () => {
+        expect(() => scheduleStore.getScheduleName(100)).not.toThrowError();
+    });
+
+    test('does not error if schedule name is undefined', () => {
+        expect(() => render(<RenameScheduleDialog index={100} open={true} />)).not.toThrowError();
```

Comment:
Boolean props can be passed with just the prop name for brevity.

```suggestion
        expect(() => render(<RenameScheduleDialog index={100} open />)).not.toThrowError();
```